server/stats.go

```diff
@@ -41,6 +41,7 @@ var statsRanking = map[rankKey]int{
 	{"WorldDevelopmentIndicators", ""}:                 3, // Population
 	{"BLS_LAUS", "BLSSeasonallyUnadjusted"}:            0, // Unemployment Rate
 	{"EurostatData", ""}:                               1, // Unemployment Rate
+	{"NYT_COVID19", "NYT_COVID19_GitHub"}:              0, // Covid
```

Comment:
Maybe I'm mistaken, but don't we need to provide the other ones too, with a lower priority?

Reply:
So by default the score is the lowest score: https://github.com/datacommonsorg/mixer/pull/245/files#diff-090a189afcd6b3645ff26daa525859f6R177-R179.